bin/build-manylinux.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+# This script builds BlazingMQ and all of its dependencies.
+#
+# Before running this script, install following prerequisites, if not present
+# yet, by copy-and-pasting the commands between `<<PREREQUISITES` and
+# `PREREQUISITES` below:
+<<PREREQUISITES
+sudo yum install -y \
+    gdb \
+    curl \
+    pkgconfig \
+    ninja-build \
```

Comment:
It's not clear to me where we use this dependency.  Is it BDE?

We could potentially use it with Google Benchmark and BlazingMQ itself as well, given both are CMake projects.